eZ/Publish/Core/Persistence/Legacy/Bookmark/Gateway.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\Core\Persistence\Legacy\Bookmark;
+
+use eZ\Publish\SPI\Persistence\Bookmark\Bookmark;
+
+abstract class Gateway
```

Comment:
Nitpick: missing very brief PhpDoc (not needed on implementations as long as here we have something)